app/workers/education_form/create_daily_spool_files.rb

```diff
@@ -147,5 +163,13 @@ def track_form_type(type, rpo)
     def stats
       @stats ||= Hash.new(Hash.new(0))
     end
+
+    def log_info(message)
+      if Flipper.enabled?(:sidekiq_create_daily_spool_file_logging)
+        log_exception_to_sentry(DailySpoolFileLogging.new(message))
+      else
+        logger.info(message)
+      end
```

Comment:
It looks like you're using our feature flipper as a configuration tool rather than for a temporary feature that we would work to remove.

Putting that aside, I don't see a reason for the conditional at all. Couldn't we just call both?

```suggestion
      log_exception_to_sentry(DailySpoolFileLogging.new(message))
      logger.info(message)
```

What are your thoughts on doing this?

Reply:
Also, it looks like `log_exception_to_sentry` [already logs to Rails' logger](https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/sentry_logging.rb#L27-L31).

Reply:
Removed feature flag and set log level to `:info` for this method